src/constants.js

```diff
@@ -18,9 +18,17 @@ const STATE = {
 };
 
 const SOURCE = {
-  STATE: { SOURCE: 'STATE' },
-  TRANSPORTER: { SOURCE: 'TRANSPORTER' },
-  CLIENT_STORAGE: { SOURCE: 'CLIENT_STORAGE' },
+  STATE: {
+    SOURCE: 'STATE',
+  },
+  TRANSPORTER: {
+    SOURCE: 'TRANSPORTER',
+    IS_READY: 'transporter',
```

Comment:
shouldnt' this be something like NAME instead of IS_READY? Then it could be used more generically?